numcodecs/vlen.pyx

```diff
@@ -250,6 +252,7 @@ class VLenBytes(Codec):
                 b = b''
             elif not PyBytes_Check(b):
                 raise TypeError('expected byte string, found %r' % b)
+            normed_values[i] = b
```

Comment:
Now when we loop through and normalize an input, we can store each normalized value for processing later